templates/views/tos.md

```diff
@@ -1,179 +1,159 @@
-# Forged Alliance Forever Terms of Service
+# Conditions d'utilisations de Forged Alliance Forever
 
-Effective date: _May 25, 2018_
+Date d'application: _25 Mai 2018_
 
-## Short Version
+## Version courte
 
-Thanks for creating an account with Forged Alliance Forever. This document outlines our Terms of Service. In short:
```

Comment:
The ToS should start with a disclaimer that the legally binding document is the English version, and a link to it directly.